### DISCUSSION TITLE
differences between dompdf and mPDF

### DISCUSSION THREAD

**Discussion by Maofree** [dompdf/dompdf#3190] [General]
Hi
in the past I used mPDF 7
Could you tell me the particular differences between dompdf and mPDF? meanwhile I see that yours weighs many megabytes less, then it seems to me that more complex html code can be used and I see more recent updates.
does it work well with php 8.2 too?
I'll start testing tomorrow

thanks
bye

**Comment by bsweeney** [comment]
I have not looked at mPDF recently so I am unable to specify the differences between the Dompdf and that product. I do believe in the past that others have stated that Dompdf, generally, provides better CSS support. On the flip side I believe mPDF provides better internationalization support.

There are a few things that may trip you up, depending on your document structure. Specifically paging of things like tables (#98) and floats. And Flexbox support is not yet available (#971).

We have been keeping up with PHP releases. So long as you're using the latest Dompdf you won't have any issues with PHP compatibility.
